docs/user-guide/SettingsFile.md

```diff
@@ -37,6 +37,72 @@ Path to your key file in a txt file.
 
 If provided and valid, RESTler will attempt to use it during the SSL handshake.
 
+### authentication: dict (default empty)
+Settings for specifying authentication. See Authentication.md for details
+
+__Token__: Can optionally provide one of {```location```, ```token_refresh_cmd```, ```module```}
+
+__location str (Default None)__: File path to a text file containing a token
+
+```json
+"authentication": {
+    "token": {
+      "location":, "/authentication_token.txt"
+      "token_refresh_interval":  300
+    }
+}
+```
+
+__token_refresh_cmd str (Default None)__: The command to execute in order to refresh the authentication token
+
+```json
+"authentication": {
+    "token": {
+      "token_refresh_cmd": "python unit_test_server_auth.py"
+      "token_refresh_interval": 300
+    }
+}
+```
+
+__module json (Default None)__: Dictionary containing settings for RESTler to invoke user-specified module to refresh the authentication token
+```json
+"authentication": {
+    "token": {
+      "module": {
+        "file": "/unit_test_server_auth_module.py",
+        "function": "acquire_token_data",
+        "data": {
+            "client_id": "client_id"
+        }
+      },
+      "token_refresh_interval": 300
+    }
+}
+```
+
+```file``` str (Default None): File path to python file containing function that returns a token
+
+```function``` str (Default "acquire_token"): Name of function in file that returns a token. The function must accept two parameters "data", a Dictionary containing the json payload specified under data, and "log" a method that will write any logs to a network auth text file
+
+```data``` json (Default None): Optional data payload to provide to function. If data is included, RESTler will attempt to call function with data as an argument
+
+__token_refresh_interval int (Default None)__: Required parameter if using token authentication. The interval between periodic refreshes of the authentication token, in seconds
+
+__Certificate__: Can optionally provide certificate for SSL handshake
```

Comment:
>__Certificate__ [](http://example.com/codeflow?start=0&length=15)

Similar to 'token', please indent this one with four hash signs and declare it (dict).  Also, please make it lowercase

Reply:
will do, will also set "Token" to lowercase for consistency